src/style.css

```diff
@@ -1,3 +1,59 @@
+/* stylelint-disable custom-property-empty-line-before */
+/* stylelint-disable comment-empty-line-before */
+/* stylelint-disable font-family-name-quotes */
 body {
   margin: 0;
+  font-family: 'Roboto', serif;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  position: relative;
+  justify-content: flex-end;
+  background-color: #fff;
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+  line-height: 60px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__link {
+  display: inline-block;
+}
+
+.nav__link:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link:hover {
+  color: var(--main-color);
+}
```

Comment:
The `<nav>` element should ideally use a list structure (`<ul>` and `<li>`) for better semantics and accessibility. Consider restructuring the navigation links within a list.